.github/workflows/build-test-with-init.yml

```diff
@@ -0,0 +1,86 @@
+name: Build with Initialization Scripts
+
+on:
+  push:
+    branches: [ main, develop ]
+  pull_request:
+    branches: [ main, develop ]
+  workflow_dispatch:
```

Comment:
We don't have `main` and `develop` branches, we have `dev` and `release/*` or `hotfix/*`